hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -365,6 +375,22 @@ void compactDB() throws StorageContainerException {
     }
   }
 
+  private void flushAndSyncDB() throws StorageContainerException {
+    try {
+      try (ReferenceCountedDB db = BlockUtils.getDB(containerData, config)) {
+        db.getStore().flushDB(true);
+        LOG.info("Container {} is synced with bcsId {}.",
```

Comment:
might be a good idea to move this log out into close() and quasiClose(). So that flushAndSyncDB is a utility routine which can, in principle, be invoked outside of close/quasiClose.